website/content/v1.8/advanced/install-kubevirt.md

```diff
@@ -0,0 +1,401 @@
+
+- creates a PVC with the requirements from either the `DataVolume` or the `dataVolumeTemplates`
+- Start a pod
+- writes temporary scratch space to local disk
+- downloads the image
+- extracts it to the temporary scratch space
+- Copies the image to the PVC
+
+Installing the CDI is very simple:
+
+```bash
+# Point to latest release
+$ export TAG=$(curl -s -w %{redirect_url} \
+https://github.com/kubevirt/containerized-data-importer/releases/latest)
+
+$ export VERSION=$(echo ${TAG##*/})
+
+# install operator
+$ kubectl create -f \
+https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+```
+
+After that, you can apply a CDI CR for the CDI operator to fully deploy CDI:
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: CDI
+metadata:
+  name: cdi
+spec:
+  config:
+    scratchSpaceStorageClass: local-path
+    podResourceRequirements:
+      requests:
+        cpu: "100m"
+        memory: "60M"
+      limits:
+        cpu: "750m"
+        memory: "2Gi"
+```
+
+This CR has some special settings that are needed for CDI to work properly:
+
+### `scratchSpaceStorageClass`
+
+This is the storage class that we installed earlier with the `local-path-provisioner`. This is needed for the CDI to write scratch space to local disk before importing the image
+
+### `podResourceRequirements`
+
+In many cases the default resource requests and limits are not sufficient for the importer pod to import the image. This will result in a crash of the importer pod.
+
+After applying this yaml file, the CDI operator is ready.
+
+## creating your first virtual machine
+
+Now it is time to create your first virtual machine in KubeVirt. This is a basic virtual machine that you can use as a boiler plate:
+
+```yaml
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: fedora-vm
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: fedora-vm
+      annotations:
+        kubevirt.io/allow-pod-bridge-network-live-migration: "true"
+        
+    spec:
+      evictionStrategy: LiveMigrate
+      domain:
+        cpu:
+          cores: 2
+        resources:
+          requests:
+            memory: 4G
+        devices:
+          disks:
+            - name: fedora-vm-pvc
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+          - name: podnet
+            masquerade: {}    
+      networks:
+        - name: podnet
+          pod: {}
+      volumes:
+        - name: fedora-vm-pvc
+          persistentVolumeClaim:
+            claimName: fedora-vm-pvc
+        - name: cloudinitdisk
+          cloudInitNoCloud:
+            networkData: |
+              network:
+                version: 1
+                config:
+                  - type: physical
+                    name: eth0
```

Comment:
Does this use the bridge adapter?

Reply:
No, this uses the pod network. Should i also add an example for bridge network configuration?

Reply:
Because we mention setting up multus and using a bridge we should probably include an example or at least a commented out section in the yaml to show how they'd use it instead of CNI.

Reply:
> Because we mention setting up multus and using a bridge we should probably include an example or at least a commented out section in the yaml to show how they'd use it instead of CNI.

I will add an example tonight. And will fix the other issues. Thanks for replying :)

Reply:
@rothgar i added an Multus example to the documentation.